src/components/pages/SavedCases/SavedCases.js

```diff
@@ -29,7 +31,7 @@ function SavedCases({ savedCases, setSavedCases, deleteBookmark }) {
       width: '5%',
     },
     {
-      title: 'PDF',
+      title: 'Download Case',
```

Comment:
How do we differentiate to the user what the difference is between "Download Case" and "Download Report"?

Reply:
I agree.
UI/UX input needed.